### ISSUE TITLE
TODO (New)

### ISSUE THREAD

**Issue by Taki21** [OperatingSytemDevs/UnnamedOS#3]
- [ ] Start Folder System
- [ ] Finish Boot Sequence
- [ ] Finish Setup on New OS installation
- [ ] Finish Graphics
- [ ] Finish Built - in Operating System Apps
- [ ] Create a Store for Downloading Apps
- [ ] Create File Formats
- [ ] Create Browser
- [ ] Create the Start Menu

@Dog2puppy 
@developerCodex this is the current TODO

**Comment by Taki21** [comment]
Now, we can start but we need to have parts. since there is 9 parts we each do 9.

**Comment by Taki21** [comment]
I will do graphics but we all need to do it since its a group thingy lol
U guys get to decide what jobs u want first :P

**Comment by Ad5001** [comment]
Maybe I could create the store or the file formats...

**Comment by Dog2puppy** [comment]
What's @developerCodex got to do with this?

**Comment by Taki21** [comment]
he wanted to join and help

**Comment by Taki21** [comment]
So:
@Ad5001 >>> File Format + Store
@Dog2puppy >>> ? + ?
@Taki21 >>> Graphics + Start Menu
@developerCodex >>> ? + ?

**Comment by Taki21** [comment]
Each of us have to do 2 parts.

**Comment by Taki21** [comment]
But note we can all help each other on any part if ur finished or bored with yer part

**Comment by Ad5001** [comment]
I'm quite not sure on how to create the app but I could make an encryption based on 7z for the apps.

**Comment by Dog2puppy** [comment]
I could do the booting screen. So that on boot, it shows a nice loading screen.

**Comment by Dog2puppy** [comment]
And I could help with graphics (Not sure how good they'll be. I use GIMP). Not sure about my second task. Maybe start menu?

**Comment by Taki21** [comment]
U could help me on start menu ya.

**Comment by Taki21** [comment]
Also, would setup be good for u?

**Comment by Dog2puppy** [comment]
Sure.

**Comment by Taki21** [comment]
Ok Now:

@Ad5001 >>> File Format + Store
@Dog2puppy >>> Boot Sequence + OS Setup
@Taki21 >>> Graphics + Start Menu
@developerCodex >>> ? + ?

**Comment by Taki21** [comment]
4 some names how bout 

CosmosOS ?

Later we can make versions like:

Cosmos Black Hole
Cosmos Supernova
Cosmos Hypernova
Cosmos Mercury
etc...

**Comment by Ad5001** [comment]
Maybe like LightOS ? Because it's light in space and is the new light of software

**Comment by HimbeersaftLP** [comment]
Some ideas:

* ImagicalOS
* NotALinux
* Takios
* TakDog5OS1
* WinTAKIdows xD
* xDOS
* LOsL

(more ideas coming soon... XD)

**Comment by Dog2puppy** [comment]
ImagicalOS. One of the apps can then be ImagicalMine, but we modify it so that ImagicalOS Users get special features!

**Comment by Ad5001** [comment]
Yeah but this means that we would have to add php ... which mean rencode it to our format...

**Comment by Taki21** [comment]
Why Imagical? Its not unique.

**Comment by Taki21** [comment]
I like CosmosOS and LightOS as a name but which one?

**Comment by Taki21** [comment]
@HimbeersaftLP  LOL XD

**Comment by Taki21** [comment]
Remember: Our OS Name must have a name with structure meaning, that in future versions we need to name it a children like I said earlier: Cosmos Supernova.
Cosmos >>> OS
Supernova >>> Version (Name)

**Comment by HimbeersaftLP** [comment]
> I like CosmosOS and LightOS as a name but which one?

I liked TakiOS best... XD

If I had to choose between CosmosOS and LightOS, I'd choose LightOS.

Version names:

* LightOS Candle
* LightOS LED
* LightOS Taki
* LightOS Sun
* LightOS Fire
* LightOS Neon
* LightOS Bright
* LightOS Glow
* LightOS Cola
* LightOS Lightweight
* LightOS Night
* LightOS Day
* LightOS Sun
* ...

**Comment by Taki21** [comment]
LOL ya ur right! xD

**Comment by Taki21** [comment]
Light it is! What do u think Everybody?

LightOS - 3 Votes

**Comment by Taki21** [comment]
If i named it TakiOS that wouldnt be fair to us lol

**Comment by HimbeersaftLP** [comment]
> If i named it TakiOS that wouldnt be fair to us lol

Then call it UnitedTakDog5001OS xD

**Comment by Taki21** [comment]
LOL xD

**Comment by Taki21** [comment]
I know! Mountain1738DewOS

**Comment by Taki21** [comment]
lol

**Comment by HimbeersaftLP** [comment]
XD call it IlluminaDOS

**Comment by Dog2puppy** [comment]
PuppyOS
The default background js puppies! XD

**Comment by Taki21** [comment]
LOL!!!

**Comment by Taki21** [comment]
Ok lets get serious :p 
shall we do LightOS? Got 3 Votes Already

**Comment by Taki21** [comment]
Wait! Windows was a DOS operating sytem at the beginning but look:

Wind**Dows**

Wind**DOS**

Wind**Dows** came from Wind**DOS** Maybe! LOL 

ILLUMINATI CONFIRMED!

https://www.google.com/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=0ahUKEwialrXqmZfOAhUIez4KHfegDWsQjRwIBw&url=http%3A%2F%2Fimgfave.com%2Fcollection%2F103484%2FIlluminati&psig=AFQjCNFFkCzLqTWZCRee5_rEGmh95De4EA&ust=146983112762562
5

https://www.google.com/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=0ahUKEwijsZD1mZfOAhXDcT4KHV71A64QjRwIBw&url=https%3A%2F%2Fwww.tumblr.com%2Ftagged%2Filluminati-gifs&psig=AFQjCNFFkCzLqTWZCRee5_rEGmh95De4EA&ust=1469831127625625

https://www.google.com/imgres?imgurl=https%3A%2F%2Fmedia.serious.io%2F969269e8d5079564%2Fserious.gif&imgrefurl=https%3A%2F%2Fserious.io%2Fgif%2F6ea74a2de8f91a06&docid=ESM8TYUA36J2zM&tbnid=YvUy1oxCxYwY7M%3A&w=607&h=607&safe=strict&bih=989&biw=1920&ved=0ahUKEwjxtIfjmZfOAhXDcT4KHV71A64QMwgsKA4wDg&iact=mrc&uact=8

https://www.google.com/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=0ahUKEwihwpuDmpfOAhVMWz4KHbQTB-IQjRwIBw&url=http%3A%2F%2Fdeadliestfiction.wikia.com%2Fwiki%2FFile%3AIlluminati.gif&psig=AFQjCNFFkCzLqTWZCRee5_rEGmh95De4EA&ust=1469831127625625

**Comment by Dog2puppy** [comment]
LightOS

**Comment by Taki21** [comment]
Ok LightOS it is!

**Comment by Taki21** [comment]
Bad NEWS!!! LightOS and CosmosOS is already made! We need a better name!

**Comment by Ad5001** [comment]
LightningOS !

**Comment by HimbeersaftLP** [comment]
But lightning is the same thing as illuminating and Illuminati is illuminated, that would mean: LightningOS = Illuminati confirmed

**Comment by Taki21** [comment]
LOL

OMG Thats Taken Too!!!

**Comment by Taki21** [comment]
ElementalOS?

**Comment by Taki21** [comment]
Yes! NOT taken!

**Comment by HimbeersaftLP** [comment]
What about MLGos

Versions:

* MLGos Doge
* MLGos Illuminati
* MLGos MountainDew
* MLGos Taki
* MLGos WOW
* MLGos GG
* MLGos triple
* MLGos m8
* MLGos 1337
* MLGos Doritos
* MLGos 333

**Comment by Ad5001** [comment]
Hum... no.
Why not like MineralOS (like MineralOS Coal, MineralOS Tin, MineralOS Gold, MineralOS Emarald)
or WoodOS (WoodOS Oak, WoodOS Birch, WoodOS Jungle)
FloatOS (floatOS Cloud, floatOS Star, floatOS Sun, floatOS Moon)
What do you think?

**Comment by developerCodex** [comment]
Sorry, about the late responce I wasn't able to be on this week. I'd be willing to do. 
Browser + Finish Built

**Comment by developerCodex** [comment]
What browser do you want me to make?

**Comment by Dog2puppy** [comment]
@Ad5001 I want both MineralOS and WoodOS.

**Comment by Dog2puppy** [comment]
WoodOS Lumberjack

**Comment by developerCodex** [comment]
@Dog2puppy We have to pick one I would like WoodOS.

**Comment by HimbeersaftLP** [comment]
What about FreshOS

Versions:

* FreshOS Water
* FreshOS Liquid
* FreshOS Cola
* FreshOS Tasty
* FreshOS A/C
* FreshOS Taki
* FreshOS Fridge
* FreshOS Freezer
* ...

**Comment by Taki21** [comment]
Mineral sounds nice. But guys search google for ur OS name if u see someone used it we cant use it

**Comment by Taki21** [comment]
FreshOS taki lol

**Comment by Taki21** [comment]
taken wtf...

**Comment by Taki21** [comment]
Omg Wood taken too wtf is this!

**Comment by Taki21** [comment]
MineralOS NOT TAKEN YES!!!

**Comment by Ad5001** [comment]
Ah :)

**Comment by Dog2puppy** [comment]
I never knew so many OSes existed besides for things like OpenComputers or ComputerCraft!

**Comment by Taki21** [comment]
So what ya guys say? MineralOs?

**Comment by Taki21** [comment]
before someone takes it :p

**Comment by developerCodex** [comment]
Yeh

**Comment by HimbeersaftLP** [comment]
NOOOOO, TakiOS everybody loves TAKIS

**Comment by Ad5001** [comment]
Me too @Dog2puppy . That's making me wondering why are we trying to make one if so many exists...

**Comment by Taki21** [comment]
some people make it but sont spread it on the internet so much and thats what i noticed with 98% of all the OS i saw. But think, we exist in many forums:

ImagicalMine
MineGamers
PocketMine
MCrafters
Etc...

We can spread it around and have many downloads for the OS also think about this:

People create websites so when people search for an OS, The website might just come up!

**Comment by Taki21** [comment]
Lol TakiOS will be too spicy for us to handle :p

**Comment by Taki21** [comment]
We also have youtube and many other social media and i know that you guys have more subs than i do XD

**Comment by GexoXYZ** [comment]
I'm looking forward to using this! You guys will be the next bill gates!

**Comment by Taki21** [comment]
xD lol thx :P

**Comment by Dog2puppy** [comment]
Will we make everything free? I like the idea of everything free.

**Comment by HimbeersaftLP** [comment]
You could also make some paid apps in the app store...

**Comment by Taki21** [comment]
nah i like free :P

**Comment by Dog2puppy** [comment]
Let's charge $0.50 so we get enough for a domain. :D Let's use github pages

**Comment by Dog2puppy** [comment]
I could do website.

**Comment by Dog2puppy** [comment]
@Taki21 Can I do the website? I use github pages.

**Comment by Taki21** [comment]
kk

**Comment by Taki21** [comment]
i can get a tk domain or something like that for now but money nah...

**Comment by Taki21** [comment]
everyone would love everything free on the OS

**Comment by Taki21** [comment]
MineralOS here we go :P

**Comment by Taki21** [comment]
i will post some pics soon

**Comment by HimbeersaftLP** [comment]
> i can get a tk domain or something like that for now but money nah...

I'd like to donate, but how o_o

**Comment by Dog2puppy** [comment]
@Taki21 So I can design the website?

**Comment by Taki21** [comment]
@Dog2puppy Ya sure :p as long as its BEAUTIFUL

@HimbeersaftLP :p

**Comment by Taki21** [comment]
We will start around next week

**Comment by Taki21** [comment]
ill make a picture of the system UI and graphics and tell me ur opinion

**Comment by GexoXYZ** [comment]
I'm curious on how you want to achieve this project, as you all seem to mostly only have php experience?

**Comment by Dog2puppy** [comment]
@GexoXYZ I think everyone here but @developerCodex knows a lot about PHP. And my MCPE life is my merging with my Discord life now... ._.

**Comment by Dog2puppy** [comment]
@Taki21 just noticed, you had a typo. You made the org Operating**Sytem**Devs

**Comment by Taki21** [comment]
wo xD

**Comment by Dog2puppy** [comment]
Lol

**Comment by Dog2puppy** [comment]
What's a sytem anyway. XD

**Comment by Taki21** [comment]
@GexoXYZ We all mostly experience with PHP but some of us do ot her languages too :p o do Java and some other weirdo languages xD

**Comment by Taki21** [comment]
![ice_screenshot_20160906-192959](https://cloud.githubusercontent.com/assets/14851895/18294718/41a540ea-7469-11e6-9052-b6cf735b6187.png)

**Comment by HimbeersaftLP** [comment]
WOW looks great!

btw can I try to design a logo?

**Comment by Taki21** [comment]
thanks :p and Sure !

**Comment by Taki21** [comment]
MineralOS

**Comment by GexoXYZ** [comment]
Wow sounds great! When will we see some code? :)

**Comment by macdja38** [comment]
Are you guys not concerned about having to load .net into a computer that has barely started booting.

**Comment by Taki21** [comment]
well cause of school, heres the Ultimate Plan Everyone:
ON WEEKENDS ONLY: May u contribute freely, if u HAVE FREE TIME on WEEKDAYS or HOLIDAYS ON WEEKDAYS we can work on it :p  

@macdja38 we will think about that. For the 1st version we wanna make it simple, but we might add more complex features later.

Our MAIN goal is to accomplish the System UI, Graphics, Internet, Settings, Apps, etc. 1st version all main simple stuff in OS. then in 2nd version we will work hard.

However, we need to create a visual before we start. Like a "house plan".

**Comment by Taki21** [comment]
I already posted the lock screen. i will need to make some other pictures so we can start Easily!

**Comment by Taki21** [comment]
Login Screen:
![ice_screenshot_20160907-134923](https://cloud.githubusercontent.com/assets/14851895/18322744/3882414a-7502-11e6-9db3-136de77aaf46.png)

**Comment by macdja38** [comment]
I'm 90% sure that's actually the windows login screen with a custom background.

**Comment by Taki21** [comment]
ya ik little different, most login screens are so similar its hard to make a new concept

**Comment by Taki21** [comment]
The Main Desktop UI will be way different

**Comment by Taki21** [comment]
Guys, we will also make transitions, (fade), so instead of appearing to the desktop, the login will fade and then the desktop fades in.

**Comment by Taki21** [comment]
i was thinking: Why dont we make this OS a mixture of Current popular OS (windows, apple) and Mobile OS?

**Comment by Taki21** [comment]
like windows and android together with our own UI?

**Comment by macdja38** [comment]
So... basically windows 8?

**Comment by HimbeersaftLP** [comment]
> So... basically windows 8?

Haha xD Ur so right

**Comment by Dog2puppy** [comment]
@Taki21 Why do we need the forums tag? Also, those examples are looking great. This is gonna be hard to do I think tho. :(

**Comment by Dog2puppy** [comment]
And you have else phobia, while I have hippopottomonstrophousquipidilaphobia.

**Comment by Taki21** [comment]
Lol. Forum tag cause lots of comments

**Comment by Taki21** [comment]
Im gonna do the graphics so it makes life easier xD

**Comment by Dog2puppy** [comment]
Lol

**Comment by Dog2puppy** [comment]
I could setup some forums.

**Comment by macdja38** [comment]
I think it might be more productive to create multiple issues, one for each thing that you guys are trying to complete, than setting up a forum for this.

**Comment by Dog2puppy** [comment]
Why? I have room left for another forum on my host.

**Comment by Taki21** [comment]
lol u guys tool this seriously :p Forum Tag was a joke xD

**Comment by Dog2puppy** [comment]
XD

**Comment by Dog2puppy** [comment]
@Taki21 I could still setup a forum tho. Even XenForo!

**Comment by Taki21** [comment]
lol but not now

**Comment by macdja38** [comment]
I mean... you guys are planning to build an OS in C#... and you are laughing at me and one of your members for taking the idea that you organize it's creation in a form as a Joke?

**Comment by Taki21** [comment]
wut? wow... u really dont get it....

**Comment by Taki21** [comment]
well we are not laughing at u ... i was making a joke that  there was too much comments to put the forum tag

**Comment by GexoXYZ** [comment]
When will we see some code? I *cannot* wait to see it! :)

**Comment by Dog2puppy** [comment]
@Ad5001 I might be able to help you a little on the App Store. We would need a MySQL database.

**Comment by Ad5001** [comment]
@Dog2puppy I will start it when the projects will be **really** started :) We cannot make ashop for an unexisting soft :)

**Comment by Dog2puppy** [comment]
@Ad5001 Ik. :P

**Comment by developerCodex** [comment]
Let's start this, please.

**Comment by Taki21** [comment]
Ok. This weekend whos free? im free on monday maybe sunday and tuesday

**Comment by Dog2puppy** [comment]
I'm free on all weekends. But I have school 8 AM - 3 PM. But at school, we do get chromebook free time if we beg our teacher enough. :P
So I could work for about 10 minutes when we get free time. Maybe even more if I'm lucky.

**Comment by Dog2puppy** [comment]
This is dead

**Comment by GexoXYZ** [comment]
No, this is Patrick!

**Comment by Taki21** [comment]
xD

**Comment by HimbeersaftLP** [comment]
http://youtu.be/YSzOXtXm8p0

**Comment by Dog2puppy** [comment]
I was excited for this. :(

**Comment by GexoXYZ** [comment]
Why not start it? 🤷

**Comment by Taki21** [comment]
ok then.... lets make a commit today!

**Comment by Dog2puppy** [comment]
@Taki21 what will the commit be?

**Comment by Taki21** [comment]
boooting?

**Comment by Dog2puppy** [comment]
Lets get started then.

**Comment by HimbeersaftLP** [comment]
What about editing the readme, it still says "To be named soon"...

**Comment by Taki21** [comment]
oh xD

**Comment by Dog2puppy** [comment]
We're never gonna make a commit. D:

**Comment by Taki21** [comment]
But how we start without the booting? i dont know dat part :(

**Comment by Taki21** [comment]
Remember our assignments? I do graphics and something else forgot wut... :p

**Comment by HimbeersaftLP** [comment]
But you can't do something ELSE, you are alergic to it!

**Comment by Taki21** [comment]
wtf

**Comment by Taki21** [comment]
ya @HimbeersaftLP thats why i do graphics :p xD
